static/openapi/est-spec.json

```diff
@@ -0,0 +1,1649 @@
+          },
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "The UTC end date and time for which activity information is retrieved from (e.g. 2022-01-01T00:00:00Z).",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Activity metrics retrieved successfully.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MetricsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/est/v1/activity/daily": {
+      "get": {
+        "tags": [
+          "Activity Metrics"
+        ],
+        "description": "Get daily activity metrics based on start and end date-time parameters.",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-PhotonIQ-CustomerID",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": "Photoniq EST CustomerID"
+          },
+          {
+            "in": "query",
+            "name": "from",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "required": true,
+            "description": "The UTC start date and time for which activity information is retrieved from (e.g. 2022-01-01T00:00:00Z)."
+          },
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "The UTC end date and time for which activity information is retrieved from (e.g. 2022-01-01T00:00:00Z).",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Activity metrics retrieved successfully.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MetricsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/activity/monthly": {
+      "get": {
+        "tags": [
+          "Activity Metrics"
+        ],
+        "description": "Get monthly activity metrics based on start and end date-time parameters.",
```

Comment:
I'm going to start a To Do list, maybe you can capture these as Jira tickets. We need to explain in the metrics topic all about these activity metrics. Usage metrics, too. Probably have separate sections in the metrics topic, or two topics.